Tasks/SonarQubePostTest/PRCA/ReportProcessor.ps1

```diff
@@ -2,6 +2,80 @@ $ProjectGuidAndFilePathMap = @{}
 $ComponentKeyAndPathMap = @{}
 $ComponentKeyAndRelativePathCache = @{}
 
+
+#region Public
```

Comment:
What about maintenance, breakdown of code to logical components? Are those not good enough reasons?